onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -128,6 +129,47 @@ Status QnnBackendManager::LoadQnnSystemLib() {
   return Status::OK();
 }
 
+void QnnLogging(const char* format,
+                QnnLog_Level_t level,
+                uint64_t timestamp,
+                va_list argument_parameter) {
+  ORT_UNUSED_PARAMETER(level);
+  ORT_UNUSED_PARAMETER(timestamp);
+
+  // Always output Qnn log as Ort verbose log
+  ::onnxruntime::logging::Capture(::onnxruntime::logging::LoggingManager::DefaultLogger(),
+                                  ::onnxruntime::logging::Severity::kVERBOSE,
+                                  ::onnxruntime::logging::Category::onnxruntime,
+                                  ::onnxruntime::logging::DataType::SYSTEM,
+                                  ORT_WHERE).ProcessPrintf(format, argument_parameter);
```

Comment:
we should also probably check whether output is enabled:

```suggestion
  const auto& logger = ::onnxruntime::logging::LoggingManager::DefaultLogger();
  const auto severity = ::onnxruntime::logging::Severity::kVERBOSE;
  const auto data_type = ::onnxruntime::logging::DataType::SYSTEM;
  if (logger.OutputIsEnabled(severity, data_type)) {
    ::onnxruntime::logging::Capture(logger,
                                    severity,
                                    ::onnxruntime::logging::Category::onnxruntime,
                                    data_type,
                                    ORT_WHERE).ProcessPrintf(format, argument_parameter);
  }
```